webapp/src/components/admin_settings/job_table/job_schedule_modal.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from 'react';
+
+import {Modal} from 'react-bootstrap';
+import {addDays} from 'date-fns';
+import {DateRange, DayPicker} from 'react-day-picker';
+
+import 'react-day-picker/dist/style.css';
+
+export type Props = {
+    show: boolean;
+    onClose: () => void;
+    onSubmit: (range: DateRange | undefined) => void;
+}
+
+const styles = {
+    buttonRow: {
+        margin: '24px 24px',
+    },
+    btn: {
+        gap: '0px',
+    },
+};
+
+const JobScheduleModal = ({show, onClose, onSubmit}: Props) => {
+    const today = new Date();
+    const defaultSelected: DateRange = {
+        from: addDays(today, -3),
+        to: today,
+    };
+    const [range, setRange] = useState<DateRange | undefined>(defaultSelected);
```

Comment:
nit: I think you should be able to do:
```suggestion
    const [range, setRange] = useState(defaultSelected);
```
which will remove the undefined as a possibility